sunspec/integrationpluginsunspec.json

```diff
@@ -41,6 +41,7 @@
                     "id": "f51853f3-8815-4cf3-b337-45cda1f3e6d5",
                     "createMethods": [ "User", "Discovery" ],
                     "interfaces": ["gateway"],
+                    "providedInterfaces": ["smartmeterproducer", "solarinverter", "energymeter", "energystorage", "battery"],
```

Comment:
battery probably shouldn't be here... energystorage is what we want.

also, smartmeterproducer I'm unsure... the solar inverter is the one from the user perspecive, I think... WDYT?